dockstore-webservice/src/main/resources/migrations.1.15.0.xml

```diff
@@ -30,48 +30,13 @@
         <addPrimaryKey columnNames="userid, entryid" constraintName="user_entry_pkey" tableName="user_entry"/>
     </changeSet>
     <changeSet id="unique_checker_ids" author="coverbeck">
-        <comment>One tool and one workflow in prod currently share the same checker workflow. The constraints later in this migration will
-            disallow that. Null out any duplicate checkers so the constraints can be created. When there are duplicates, prefer the
-            workflow table, then the apptool table, for the table that "keeps" the checker.
+        <comment>One tool and one workflow in prod currently share the same checker workflow.  Null out any duplicate checkers.
+            When there are duplicates, prefer the workflow table, then the apptool table, for the table that "keeps" the checker.
         </comment>
         <sql dbms="postgresql">
             UPDATE tool SET checkerid=NULL where checkerid in (SELECT checkerid FROM workflow UNION SELECT checkerid FROM apptool);
             UPDATE apptool SET checkerid=NULL where checkerid in (SELECT checkerid FROM workflow UNION SELECT checkerid FROM tool);
         </sql>
-        <sql dbms="postgresql">
```

Comment:
I think this will cause our nightly automatic QA migrations to fail, but since this hasn't made into prod, that might be OK. After this merged and deployed to QA, we'll have to do a restore from prod to start the migrations from scratch from 1.14.

The other option is to create a new change set at the end that drops and removes the constraints, but I suppose there's some risk there in prod if something goes wrong between the 2 changesets.

Reply:
Yeah, it's probably better to remove the constraints via a new changeset at the end, should be relatively bulletproof.